runtime-modules/project-token/src/types.rs

```diff
@@ -569,6 +575,57 @@ where
     }
 }
 
+/// Represents token's bonding curve with linear pricing function y = ax + b
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Default, Encode, Decode, TypeInfo, Clone, Debug, Eq, PartialEq, MaxEncodedLen)]
+pub struct BondingCurve {
+    /// Slope parameter : a
+    pub slope: Permill,
+
+    /// Intercept : b
+    pub intercept: Permill,
```

Comment:
For `b` the problem is even more pronounced, because this is the main parameter which allows controlling the initial price. If it cannot be more than 1, then creator has almost no control over this.

I don't have an issue with this being fixed in the next PR, but I don't agree it's not a problem (this applies to both `a` and `b` actually)